docs/SUMMARY.md

```diff
@@ -1,15 +1,15 @@
 # Table of contents
 
-- [Introduction](../README.md)
+- [Introduction](./readme.md)
```

Comment:
@Amruta-Ranade has pointed out this whole file is an artifact of the gitbook days, so imma murder it